### DISCUSSION TITLE
Roadmap for demo extensions

### DISCUSSION THREAD

**Discussion by sgbaird** [sparks-baird/self-driving-lab-demo#77] [Demo extensions and design]
## Liquid-based color matching
The demo I'm planning to design next (with similar instructions, bill of materials, etc.) is a color-matching demo using red/green/blue food coloring dissolved in water and housed in medical IV-style bags, pumped via several peristaltic pumps, with light matching via the existing AS7341 light sensor. I think I can design that one for under $200 and under 2 hours of setup time.

- https://github.com/sparks-baird/self-driving-lab-demo/discussions/89

## pH optimization of chemicals
From there, I want to purchase a pH sensor and replace the red/green/blue food coloring dyes with other chemicals (I forget which ones specifically, but there are a few manuscripts that do pH optimization). Still probably in the < $200 and < 2 hrs setup, since the only additional materials are the pH sensor, cabling for the pH sensor, and the chemicals.

- https://github.com/sparks-baird/self-driving-lab-demo/discussions/223

## Powder and solid handling via robotics/computer vision
After that, I want to move to some robotics and computer vision tasks, and if all goes well, implement powder and solid handling. This will probably be ~$600-$800, not sure about the setup time yet, but probably 2-10 hours.

- https://github.com/sparks-baird/self-driving-lab-demo/discussions/153

## 
See also https://github.com/sparks-baird/self-driving-lab-demo/discussions/53 for additional ideas

**Comment by jakobzeitler** [comment]
Super keen to see how the powder handling works out!
